### DISCUSSION TITLE
Изменение шаблона по умолчанию для таблицы циклов (или менеджмент таких шаблонов)

### DISCUSSION THREAD

**Discussion by simplepersonru** [totumonline/totum-ru-issues-and-discussions#387] [Есть идея, а что если...]
Сейчас в таблице циклов **ТЦ** автоматом создаются 2 поля: Доступ и Кнопка в цикл, причем Кнопка видима не только создателю.

Минусы:
1. Допустим, что в **ТЦ** будет дублироваться много информации из расчетных таблиц в цикле **РВЦ**. Кнопка занимает полезное место своей площадью
2. По сути это поле не несет визуально полезной информации, когда мы анализируем **ТЦ** в отрыве от отдельных **РВЦ**

Предложение:
Как правило в **РВЦ** есть некий **идентификатор**, это или строка или совокупность каких-то строк \ чисел... на худой конец id **РВЦ**. Под этот **идентификатор** мы создадим в таблице циклов колонку, в которой будет код действия -> по клику -> оформить как url.  Непосредственно код - тот который щас в кнопке
![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/71624171/a1d84ff7-ab72-4424-8050-36ef9d4b001d)

Мне субьективно кажется это более нативным переходом к **РВЦ** и такой подход я видел в подавляющем количестве схожих по функционалу систем

**Comment by ttm-alexeypolunin** [comment]
Мы не поняли в чем суть вопроса - тк вы перенастроили как вам нужно для конкретной таблицы циклов и все. Лично нам нравится кнопка :)

**Reply by simplepersonru** [reply]
Это не вопрос, топик для предложений, насколько я понимаю :) ответ понял
